File: app/src/main/java/com/anhhoang/popularmovies/model/ReviewResponse.java
package com.anhhoang.popularmovies.model;
/*
 * Copyright (C) 2013 The Android Open Source Project
 */

import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by AnhHo on 3/20/2017.
 */

public class ReviewResponse {
    @SerializedName("id")
    private Integer id;
    @SerializedName("page")
    private Integer page;
    @SerializedName("results")
    private List<Review> results;
    @SerializedName("total_pages")
    private Integer totalPages;
    @SerializedName("total_results")
    private Integer totalResults;
}

File: MovieApp/app/src/main/java/com/anhhoang/popularmovies/model/ReviewResponse.kt
package com.anhhoang.popularmovies.model

/*
 * Copyright (C) 2013 The Android Open Source Project
 */

import com.google.gson.annotations.SerializedName

/**
 * Created by AnhHo on 3/20/2017.
 */

class ReviewResponse(
        @SerializedName("id")
        val id: Int = 0,
        @SerializedName("page")
        val page: Int = 0,
        @SerializedName("results")
        val results: List<Review>? = null,
        @SerializedName("total_pages")
        val totalPages: Int = 0,
        @SerializedName("total_results")
        val totalResults: Int = 0
)
